app/views/exercises/_exercises.html.erb

```diff
@@ -1,16 +1,26 @@
-<div style="display:flex;">
-  <div style="margin-right: 20px;">
+<div class="serie_content space" data-controller="green_set">
+  <div class="leftpart">
     <h3><%= exercise.repetitions %></h3>
-  </div>      
-  <div style="margin-right: 20px;">
     <h3><%= exercise.weight %></h3>
   </div>
-  <div style="margin-right: 20px;">
-    <%= link_to 'Edit', edit_routine_series_exercise_path(@routine, serie, exercise), class: 'btn__actions' %>
+  <div class="rightpart">
+    <div class="checkbox-wrapper-44 miCheckbox">
```

Comment:
Maybe change micheckbox to something more easy to understand when you see it 
What if there's another checkbox would it be yourcheckbox?
Remember to have good variable naming to have an easier time understanding your code